Content.Shared/Doors/Systems/SharedDoorSystem.cs

```diff
@@ -536,9 +541,25 @@ public override void Update(float frameTime)
 
             if (door.NextStateChange.Value < time)
                 NextState(door, time);
+
+            if (door.State == DoorState.Closed &&
+                TryComp<PhysicsComponent>(door.Owner, out var doorBody))
+            {
+                // If something bumped into us during closing then start to re-open, otherwise, remove it from active.
+                if (doorBody.ContactCount == 0)
+                {
+                    _activeDoors.Remove(door);
+                    continue;
+                }
+
+                // Access isn't predicted so
+                CheckDoorBump(door, doorBody);
```

Comment:
Actually maybe just change this to
```suggestion
                _activeDoors.Remove(door);
                CheckDoorBump(door, doorBody);
```
That way it doesn't keep checking if some non-bumping entity gets stuck in the door. If a new one starts to collide, it should still open via the start collide event.

Reply:
Actually I don't think the fix is so simple; denying also adds it to active and is used for tracking the state change so it still spams.

Reply:
You mean that when a door-bumper with insufficient permissions gets stuck colliding to the door while closing, it gets stuck cycling between deny and closed?

Reply:
Yeah